backend/app/notify/email.go

```diff
@@ -164,35 +165,55 @@ func (e *Email) setTemplates() error {
 	return nil
 }
 
-// Send email about comment reply to Request.Email if it's set,
-// also sends email to site administrator if appropriate option is set.
+// Send email about comment reply to Request.Emails and Request.AdminEmails
+// if they're set.
 // Thread safe
 func (e *Email) Send(ctx context.Context, req Request) error {
-	if req.Email == "" {
-		// this means we can't send this request via Email
-		return nil
-	}
 	select {
 	case <-ctx.Done():
-		return errors.Errorf("sending message to %q aborted due to canceled context", req.Email)
+		return errors.Errorf("sending email messages about comment %q aborted due to canceled context", req.Comment.ID)
 	default:
 	}
 
-	if req.parent.User.ID == req.Comment.User.ID && !req.ForAdmin {
-		// don't send anything if if user replied to their own comment
-		return nil
+	result := new(multierror.Error)
+
+	// send user notifications
+	for _, email := range req.Emails {
+		email := email
```

Comment:
unless I missed something I see no reason for this as the flow with `email` seems to be synced, not  captured for goroutine